### PR TITLE
Just clean some code

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

**Comment by nono** [comment]
Travis build is red but it seems it's not related to this PR but it's from before. I can see that #207 has already a red build.

File: server/lib/encryption.coffee [hunk 1]
```diff
@@ -98,28 +95,25 @@ updateKeys = (oldKey, password, encryptedslaveKey, callback) ->
...
 exports.encrypt = (password) ->
     if password? and process.env.NODE_ENV isnt "development"
-        if masterKey? and slaveKey?
+        if slaveKey?
             newPwd = cryptoTools.encrypt slaveKey, password
             return newPwd
         else
             sendMail()
-            err = new Error "master key and slave key don't exist"
+            err = new Error "slave key does not exist"
```
Comment (frankrousseau) [inline server/lib/encryption.coffee]: *doesn't* is prefered over *does not*.

**Comment by nono** [comment]
Well, I think it's coming from the release of node 0.10.42 and the upgrade of http-parser: https://nodejs.org/en/blog/release/v0.10.42/

**Comment by nono** [comment]
There were two fixes for http-parser: [one for node 0.10](https://github.com/nodejs/node/pull/5242) and [one for node 0.12](https://github.com/nodejs/node/pull/5241). I really hope that the next versions of node 0.10 & 0.12 will fix our travis build.

**Comment by nono** [comment]
And the build is green! New versions of node 0.10 and node 0.12 have been released (with a fix for http-parser), I've relaunched the build for them and it works. This PR can be reviewed/merged.

**Pull request merged**